Helper/PaymentMethods.php

```diff
@@ -389,7 +397,9 @@ protected function showLogosPaymentMethods($paymentMethods, array $paymentMethod
             );
 
             foreach ($paymentMethods as $paymentMethod) {
-                $paymentMethodCode = $paymentMethod['type'];
+                $paymentMethodCode = array_search($paymentMethod['type'], self::METHODS_WITH_BRAND_LOGO) !== false
```

Comment:
Not super important but I think `In_array` is a cleaner way to do this
```suggestion
                $paymentMethodCode = in_array($paymentMethod['type'], self::METHODS_WITH_BRAND_LOGO)
```